Tasks/Gradle/gradletask.ts

```diff
@@ -173,9 +173,6 @@ async function run() {
 
         // Set working directory
         let workingDirectory: string = tl.getPathInput('cwd', false, true);
-        if (!workingDirectory) {
```

Comment:
The 'cwd' is a filePath, so it will never be empty in a task. It will default to $(System.WorkingDirectory) if not set by a user. This code path was only getting hit by L0 tests, so updated the tests.